yellowbrick/features/pca.py

```diff
@@ -20,23 +20,23 @@
 # NOTE: must import mplot3d to load the 3D projection
 import numpy as np
 import matplotlib.pyplot as plt
-import mpl_toolkits.mplot3d  # noqa
 from mpl_toolkits.axes_grid1 import make_axes_locatable
 
-from yellowbrick.features.base import MultiFeatureVisualizer
-from yellowbrick.style import palettes
-from yellowbrick.exceptions import YellowbrickValueError
+from yellowbrick.features.projection import ProjectionVisualizer
+from yellowbrick.exceptions import YellowbrickValueError, NotFitted
 
 from sklearn.pipeline import Pipeline
 from sklearn.decomposition import PCA
 from sklearn.preprocessing import StandardScaler
+from sklearn.exceptions import NotFittedError
 
 
 ##########################################################################
 # 2D and 3D PCA Visualizer
 ##########################################################################
 
-class PCADecomposition(MultiFeatureVisualizer):
+
+class PCADecomposition(ProjectionVisualizer):
```

Comment:
I would like to rename this to `PCA`, I think it fits in better with the way we name visualizers. At the bottom of this file you can alias to the old name for backward compatibility:

```python
# Alias for PCA
PCADecomposition = PCA
```

Reply:
I will change this!